tests/kernel/smp/prj.conf

```diff
@@ -2,3 +2,5 @@ CONFIG_ZTEST=y
 CONFIG_SMP=y
 CONFIG_TRACE_SCHED_IPI=y
 CONFIG_POLL=y
+CONFIG_NUM_PREEMPT_PRIORITIES=32
+CONFIG_NUM_COOP_PRIORITIES=32
```

Comment:
Seems like this is more than you need?  Also pedantry: the idle thread is not included in the preemptive count (for dumb historical reasons -- it used to be a special cased priority), so this is going to create a configuration with 65 total priorities, not 64, and thus require an extra word of storage in the MULTIQ scheduler bitmask.